UMLEditor/UMLEditor/Views/ClassBox.axaml

```diff
@@ -0,0 +1,103 @@
+                    VerticalAlignment="Center"
+                    FontWeight="Bold"/>
+            </ScrollViewer>
+            <Button Grid.Column="5" Classes="ControlButton" ToolTip.Tip="Rename This Class">
+                <Image Classes="ControlIcon" Source="../Assets/CustomIcons/PencilButton.png"></Image>
+            </Button>
+            <Button Grid.Column="6" Classes="ControlButton" ToolTip.Tip="Add A Relationship">
+                <Image Classes="ControlIcon" Source="../Assets/CustomIcons/DoubleArrowButton.png"></Image>
+            </Button>
+            <Button Grid.Column="7" Classes="ControlButton" ToolTip.Tip="Delete This Class">
+                <Image Classes="ControlIcon" Source="../Assets/CustomIcons/TrashCanButton.png"></Image>
+            </Button>
+        </Grid>
+        <!-- FIELDS TITLE ROW -->
+        <Grid 
+            RowDefinitions="*" 
+            ColumnDefinitions="* * * * * * * *" 
+            HorizontalAlignment="Stretch" 
+            Background="#D85A00">
+            <ScrollViewer Grid.ColumnSpan="7" HorizontalScrollBarVisibility="Auto">
+                <Label
+                    Content="Fields" 
+                    FontStyle="Italic"
+                    FontSize="16" 
+                    VerticalAlignment="Center"
+                    Margin="5 5 0 5 "/>
+            </ScrollViewer>
+            <Button Grid.Column="7" Classes="ControlButton" ToolTip.Tip="Create New Field">
+                <Image Classes="ControlIcon" Source="../Assets/CustomIcons/PlusButton.png"></Image>
+            </Button>
+        </Grid>
+        <!-- FIELDS LISTING AREA -->
+        <ScrollViewer HorizontalAlignment="Stretch" MaxHeight="70" VerticalAlignment="Stretch" VerticalScrollBarVisibility="Auto">
+            <StackPanel Name="FieldsArea">
+                
+            </StackPanel>
+        </ScrollViewer>
+        <!-- METHODS TITLE ROW -->
+        <Grid 
+            RowDefinitions="*" 
+            ColumnDefinitions="* * * * * * * *" 
+            HorizontalAlignment="Stretch" 
+            Background="#D85A00">
+            <ScrollViewer Grid.ColumnSpan="7" HorizontalScrollBarVisibility="Auto">
+                <Label
+                    Content="Methods" 
+                    FontStyle="Italic"
+                    FontSize="16" 
+                    VerticalAlignment="Center"
+                    Margin="5 5 0 5 "/>
+            </ScrollViewer>
+            <Button Grid.Column="7" Classes="ControlButton" ToolTip.Tip="Create New Method">
+                <Image Classes="ControlIcon" Source="../Assets/CustomIcons/PlusButton.png"></Image>
+            </Button>
+        </Grid>
+        <!-- METHODS LISTING AREA -->
+        <ScrollViewer HorizontalAlignment="Stretch" VerticalAlignment="Stretch"  MaxHeight="140"  VerticalScrollBarVisibility="Auto">
+            <StackPanel Name="MethodsArea">
+                
+            </StackPanel>
+        </ScrollViewer>
+    </StackPanel>
+</UserControl>
```

Comment:
As we're starting to get more and more gritty with our axaml code, I'm thinking we should do some kind of documentation pass so that we can better define what sections of our front-end code is doing.  Just a recommendation but I think it could be beneficial to the readability of our project.